service/src/test/kotlin/app/cash/backfila/service/BackfillRunnerTest.kt

```diff
@@ -116,7 +116,7 @@ class BackfillRunnerTest {
       try {
         // We should only get numthreads=1 calls in parallel, then it must wait for more room.
         assertThat(fakeBackfilaClientServiceClient.runBatchRequests.receive()).isNotNull()
-        yield()
+        delay(1) // allow the other coroutine scope to complete TODO: yield to right scope
```

Comment:
yield is on the wrong coroutine scope. I'll refactor this later. 

I'm wondering if the delay is better than poking the scope though to the tests.